### PR TITLE
Bitcoin primer update

### PR DESCRIPTION
**Pull request by Chainomics**
# Description
I have added new slides to the primer slide deck, changed some of the existing ones, and added new figures and images. @Rothj, please note, that the requested figure for the asymmetric cryptography lecture is also part of this PR. I used the figure in this primer and would like to quickly revisit it at a later stage, i.e., in the intro to pubkey crypto. 

## Type of Change
- [X] Typo fix (Changes that fix spelling errors)
- [X] Minor feature (Small changes to slides or other files)
- [X] Major feature (New files or assets, large changes to files)

# Checklist:
- [X] I have performed a self-review of my own changes
- [X] I have made sure that any changed LaTeX files compile properly
- [X] I have assigned at least one reviewer

### CODE REVIEW

**Comment by golmit** [comment]
I fixed one typo in commit 5c21d41.

**Review by golmit — APPROVED** [review]
Looks good to me!

File: slides/bitcoin-primer.tex [hunk 1]
```diff
@@ -146,9 +169,9 @@
 \uncover<3->{
 	\item{Miner chooses a subset of the transactions from its queue (mempool) to create a block, as long as they...}
 	\begin{itemize}
-		\item{\dots are legitimate.}
-		\item{\dots do not conflict with other transactions.}
-		\item{\dots do not exceed block size limitation.}
+		\item{\dots ...are legimitate.}
+		\item{\dots ...do not conflict with other transactions.}
+		\item{\dots ...do not exceed block size limitation.}
```
Comment (dariothuerkauf) [inline slides/bitcoin-primer.tex:174]:
```suggestion
		\item{\dots do not exceed block size limitation.}
```

File: slides/bitcoin-primer.tex [hunk 2]
```diff
@@ -146,9 +169,9 @@
 \uncover<3->{
 	\item{Miner chooses a subset of the transactions from its queue (mempool) to create a block, as long as they...}
 	\begin{itemize}
-		\item{\dots are legitimate.}
-		\item{\dots do not conflict with other transactions.}
-		\item{\dots do not exceed block size limitation.}
+		\item{\dots ...are legimitate.}
+		\item{\dots ...do not conflict with other transactions.}
```
Comment (dariothuerkauf) [inline slides/bitcoin-primer.tex:173]:
```suggestion
		\item{\dots do not conflict with other transactions.}
```

**Pull request merged**